doc/ledger3.texi

```diff
@@ -4616,13 +4616,49 @@ from the file above.
 @findex --rich-data
 
 The @command{convert} command accepts three options. They are
-@option{--invert} which inverts the amount field,
-@option{--account @var{STR}} which you can use to specify the account to
-balance against, and @option{--rich-data} which stores
-additional metadata as tags. There is, for example,
-a UUID field. If an entry with the same UUID tag is already included in
-the normal ledger file (specified via @option{--file @var{FILE} (-f)} or
-via the environment variable @env{LEDGER_FILE}) this entry will not be
+@option{--invert} which inverts the amount field, @option{--account
+@var{STR}} which you can use to specify the account to balance against,
+and @option{--rich-data} which stores additional metadata as tags.
+
+Using the two first lines of the above csv file,
+
+@smallexample @c input:DD85579
+767718,12/13/2011,"Withdrawal","ACE HARDWARE 16335 S HOUGHTON RD",-8.80,,00001640.04,,
+767406,12/13/2011,"Withdrawal","ACE HARDWARE 16335 S HOUGHTON RD",-1.03,,00001648.84,,
+@end smallexample
+
+and launching the below command,
+
+@smallexample @c command:DD85579
+$ ledger convert download.csv --input-date-format "%m/%d/%Y" \
+  --invert --account Assets:MyBank --rich-data
```

Comment:
Add `--now 2015/02/13` to the options, otherwise the test will fail, since the convert command will generate the `Imported` tag with the date the test is run.